data/schema/gui/widget_instances.cfg

```diff
@@ -428,6 +460,7 @@
         {DEFAULT_KEY "label" t_string ""}
         {DEFAULT_KEY "hint_text" t_string ""}
         {DEFAULT_KEY "hint_image" string ""}
+        {DEFAULT_KEY "editable" bool true}
```

Comment:
This one should go to the 1.19 branch only.

Reply:
Removed. Missed that.